src/OpenTelemetry.AutoInstrumentation/Configuration/EnvironmentConfigurationHelper.cs

```diff
@@ -31,7 +31,8 @@ internal static class EnvironmentConfigurationHelper
     {
         [Instrumentation.HttpClient] = builder => builder.AddHttpClientInstrumentation(),
         [Instrumentation.AspNet] = builder => builder.AddSdkAspNetInstrumentation(),
-        [Instrumentation.SqlClient] = builder => builder.AddSqlClientInstrumentation()
+        [Instrumentation.SqlClient] = builder => builder.AddSqlClientInstrumentation(),
+        [Instrumentation.MongoDb] = builder => builder.AddSource("MongoDB.Driver.Core.Extensions.DiagnosticSources")
```

Comment:
hmm, wondering what we should add? Instrumentation is source code based but it would not work without bytecode modifications 🤔

Reply:
I'm leaning towards calling it byte-code instrumentation, because it will only work if the profiling APIs are enabled. The would need to provide their own code to do the bootstrapping for it to be fully considered source-code instrumentation.